python/main.py

```diff
@@ -23,10 +26,38 @@ def root():
     return {"message": "Hello, world!"}
 
 @app.post("/items")
-def add_item(name: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...)):
     logger.info(f"Receive item: {name}")
+
+    # json format to be written
+    items_list = {"items" : []}
+
+    # if item.json exsits, load the data first
+    if os.path.isfile('items.json') == True:
+        with open('items.json') as item_json_file:
+            items_list = json.load(item_json_file)
+
+    # add new item
+    new_item = {"name": name, "category": category}
+    items_list["items"].append(new_item)
+
+    # Writing to item.json
+    with open("items.json", "w") as item_json_file:
+        json.dump(items_list, item_json_file, indent = 4)
+
     return {"message": f"item received: {name}"}
 
+@app.get("/items")
+def get_items():
+    items_list = 'No items found'
+
+    # if item.json exsits, load the data
+    if os.path.isfile('items.json') == True:
+        with open('items.json') as item_json_file:
+            items_list = json.load(item_json_file)
+
+    return items_list
```

Comment:
This function either returns a json file as intended or a string `No items found`, which might confuse the client side.

Check the example below from `get_image` on raising an HTTP exception. 
https://github.com/mercari-build/mercari-build-training-2022/blob/d59a374a26e8e89db05de96144d6c776ef6edeca/python/main.py#L36

Reply:
Thank you.  HTTP exception was included in line 59.